gitpractice/javascript/task12.js

```diff
@@ -0,0 +1,11 @@
+//http://www.codewars.com/kata/can-you-keep-a-secret
+function createSecretHolder(secret) {
+
+    return {
+      Secret:secret,
+      getSecret:function(){
+        return this.Secret;},
+     setSecret:function(secret){
+     this.Secret=secret;}
+    }
```

Comment:
look into the indentation of this code, use same indentation rules through out the code.
tab space 4 is prefered normally